addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -454,6 +454,7 @@
                 <field name="employee_id" filters="1"/>
                 <field name="is_hatched" invisible="1" />
                 <field name="is_striked" invisible="1"/>
+                <field name="state" invisible="1"/>
```

Comment:
thanks for your work, small question to ask 

what is reason behind this change? if it is necessary for your enterprise PR you can mention it in a commit message .